src/ext/ca/wixca/dll/secureobj.cpp

```diff
@@ -690,7 +703,7 @@ extern "C" UINT __stdcall ExecSecureObjects(
         //
         ea.grfAccessMode = SET_ACCESS;
 
-        if (0 == lstrcmpW(L"CreateFolder", pwzTable))
+        if (dwAttributes & SECURE_OBJECT_ATTRIBUTE_INHERITABLE)
```

Comment:
Need to update WixCop to handle the migration where `PermissionEx` element needs `Inheritable` attribute added by default when under a `CreateFolder` element to match WiX v3.x behavior.